langmeta/langmeta/shared/src/main/scala/org/langmeta/semanticdb/Symbol.scala

```diff
@@ -80,31 +84,42 @@ object Symbol {
         buf.toString
       }
 
-      def parseGlobal(curr: Symbol): Symbol = {
+      def parseSingle(owner: Symbol): Symbol = {
+        def global(signature: Signature): Symbol.Global = {
+          if (owner == Symbol.None && signature != Signature.Term("_root_")) {
```

Comment:
What do you think about  parsing `_root_.a.` into the same structure as `a.`? It could be implemented as simply as `.stripPrefix("_root_")`.

This would come at the price that to parse a top-level package named `_root_` you would need to write `_root_._root_.a`. However, any person who creates a top-level package named `_root_` is welcome to suffer the consequences. The backward compatibility with all previously hand-written symbols is worth it IMO

Reply:
Good idea. I'll update the spec and the implementation.

Reply:
https://github.com/scalameta/scalameta/issues/1424